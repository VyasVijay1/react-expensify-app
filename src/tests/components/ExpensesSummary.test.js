import {ExpensesSummary} from '../../components/ExpensesSummary'
import React from 'react'
import {shallow} from 'enzyme'
import toJSON from 'enzyme-to-json'

test('should render Expense summary with no expense', () => {
    const wrapper = shallow(<ExpensesSummary/>)
    expect(toJSON(wrapper)).toMatchSnapshot()
})

test('should render Expense summary with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235}/>)
    expect(toJSON(wrapper)).toMatchSnapshot()
})

test('should render Expense summary with multiple expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={23512340987}/>)
    expect(toJSON(wrapper)).toMatchSnapshot()
})