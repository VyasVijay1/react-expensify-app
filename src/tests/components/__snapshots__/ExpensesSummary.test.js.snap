// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense summary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`should render Expense summary with multiple expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $235,123,409.87
  </h1>
</div>
`;

exports[`should render Expense summary with no expense 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $0.00
  </h1>
</div>
`;
